tests/parsers/text_parser.py

```diff
@@ -44,9 +44,6 @@ def testConstantIPv4(self):
     self.assertTrue(
         text_parser.PyparsingConstants.IPV4_ADDRESS.parseString('1.1.34.2'))
 
-    with self.assertRaises(pyparsing.ParseException):
```

Comment:
This test (and the one on line 60) were removed because both these strings contain valid IP addresses: 1.1.34.25 and 10.52.34.25.

Reply:
this changes the behavior of the existing parsers, especially where this is currently used for verification

can you check which text file parsers might be affected by this

Reply:
The IP_ADDRESS, IPV4_ADDRESS and IPV6_ADDRESS are used in:
iis.py
winfirewall.py
apache_access.py
ssh.py

None of these use the constants for verification, they're used as part of larger pyparsing grammars.